File: MPI/mpicheck/tests/functions/1.0/MPI_Waitany.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    MPI_Request var_1[2];
    int *var_2;
    MPI_Status *var_3;
    int ret;
    /* calls */
    ret = MPI_Waitany(var_0, var_1, var_2, var_3);
    ret = PMPI_Waitany(var_0, var_1, var_2, var_3);
    return 0;
}

File: MPI/mpicheck/tests/functions/1.0/MPI_Waitany.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1(10)
       INTEGER var_2
       INTEGER var_3(10)
       INTEGER var_4
        call mpi_waitany(var_0, var_1, var_2, var_3, var_4)
       call pmpi_waitany(var_0, var_1, var_2, var_3, var_4)
        end program main
    